types/request.d.ts

```diff
@@ -132,21 +132,16 @@ declare class Request {
      * `Request` parameters including the URL, HTTP method and headers, and others.
      */
     constructor(options?: RequestOptions);
-    /** @type {string} */
-    id: string;
-    /** @type {string} */
+    id: any;
```

Comment:
if you remove the `@type {string}`, the generated typings will be `any` instead of string, because it's being assigned inside the constructor. that's why they need to be left in the function body (`@property` is not enough). this comment is pertinent to the other places that things became `any`. you can use `@ignore` so it doesn't go to the docs, but it's needed for the typings

Reply:
I know, but it broke the docs. It could've been fixed with `@ignore`, but I just could not stand looking at

```js
/**
 * @type {string}
 * @ignore
 */
this.property = property;
/**
 * @type {string}
 * @ignore
 */
this.property2 = property2;
/**
 * @type {string}
 * @ignore
 */
this.property3 = property3;
```

We need to figure out a better way to do this. I mean, I'm all for TypeScript support, but not for the detriment of readability of the actual code.